### PR TITLE
Add barebones search form view

### PR DESCRIPTION
**Pull request by florimondmanca**
## Description

Create a barebones form to allow the user to enter a series they want to search.

- The view is mounted on the root URL `/`, which is probably temporary.
- Clicking the "search" button redirects to that same URL, which is also probably temporary.

## Preview

screen shot 2018-10-12 at 09 53 38

## Next steps

Implement actual search: use the TVDB API to perform the search and create a new view to display the results.

## Additional context

Docs of Django concepts used here:

- [Docs on FormView](https://docs.djangoproject.com/fr/2.1/ref/class-based-views/generic-editing/#formview) and on [class-based views](https://docs.djangoproject.com/en/2.1/topics/class-based-views/)
- [Docs on building Django forms](https://docs.djangoproject.com/fr/2.1/topics/forms/#building-a-form)
- [API reference on Django templates](https://docs.djangoproject.com/fr/2.1/topics/templates/)

### CODE REVIEW

File: series/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from series import views
+
+urlpatterns = [
+    path('', views.SearchSeriesView.as_view(), name='search-series'),
```
Comment (BenoitECP) [inline series/urls.py]: Use "search_series" because "_" is already used in the file names.

Comment (florimondmanca) [reply series/urls.py]: Fine, I'll change that. :)

Comment (florimondmanca) [reply series/urls.py]: @BenoitECP Updated in new version. I also added a "conventions" section in the README to document conventions like this one.

**Comment by florimondmanca** [comment]
[Créer la vue/le template pour la page avec le formulaire de recherche](https://trello.com/c/kMGJXh5W/29-cr%C3%A9er-la-vue-le-template-pour-la-page-avec-le-formulaire-de-recherche)

**Pull request merged**